frontend/src/components/elements/PlotlyChart/CustomTheme.tsx

```diff
@@ -369,16 +352,16 @@ export function applyStreamlitThemeTemplateLayout(
       title: {
         font: {
           size: fontSizes.twoSmPx,
-          color: getGray70(theme),
+          color: theme.genericColors.axisText,
```

Comment:
Hm this is technically not necessarily a correct variable name but at least we're reusing?